tests/foreman/api/test_errata.py

```diff
@@ -55,6 +54,8 @@
 
 CUSTOM_REPO_URL = FAKE_6_YUM_REPO
 CUSTOM_REPO_ERRATA_ID = FAKE_2_ERRATA_ID
+DISTRO_RHEL6 = settings.distro.image_el6
```

Comment:
Looks like using `DISTRO_RHEL6 = settings.distro.image_el6` as a global variable also does not work, looks like it worked earlier as I missed to remove the value from the constants.py file, but there was no reference to it in the tests.